db/migrate/20250220115944_drop_shaphyr_tables.rb

```diff
@@ -0,0 +1,63 @@
+class DropShaphyrTables < ActiveRecord::Migration[7.2]
+  def up
+    drop_table :saphyr_chips, if_exists: true
+    drop_table :saphyr_enzymes, if_exists: true
+    drop_table :saphyr_flowcells, if_exists: true
+    drop_table :saphyr_libraries, if_exists: true
+    drop_table :saphyr_requests, if_exists: true
+    drop_table :saphyr_runs, if_exists: true
+  end
+
+  def down
+    create_table "saphyr_chips", charset: "utf8mb3", force: :cascade do |t|
```

Comment:
Think down can just be 
```
  def down
    fail ActiveRecord::IrreversibleMigration
  end
```
As this isn't really reversible because of the data loss.